packages/jellyfish-transaction/__tests__/script/dftx/dftx_pool/PoolSwap.test.ts

```diff
@@ -7,15 +7,15 @@ import { CPoolSwap, PoolSwap } from '../../../../src/script/dftx/dftx_pool'
 
 it('should bi-directional buffer-object-buffer', () => {
   const fixtures = [
-    '6a4c4f446654787317a914c34ca9c54dc87e7e875b212ec6ba0704be3de587870000d6117e0300000017a914c34ca9c54dc87e7e875b212ec6ba0704be3de5878702ffffffffffffff7fffffffffffffff7f',
-    '6a4c4f446654787317a914635794bb3db157b6dac4dcb7467710a10f03c6ac8700809698000000000017a914582328ee9beebc3fb5844964302cc0fdbb27e04c8702ffffffffffffff7fffffffffffffff7f',
+    '6a4c4f446654787317a914c34ca9c54dc87e7e875b212ec6ba0704be3de587870000d6117e0300000017a914c34ca9c54dc87e7e875b212ec6ba0704be3de58787022fe28e7915000080ffcf430300000000',
+    '6a4c4f446654787317a914635794bb3db157b6dac4dcb7467710a10f03c6ac8700809698000000000017a914582328ee9beebc3fb5844964302cc0fdbb27e04c87022fe28e7915000080ffcf430300000000',
```

Comment:
Do we need to override these? The fixtures are taken from the mainnet.

Reply:
The original has the maxPrice.fraction = fffffffffffffff7f which exceeded 1e8. So I think they are derived from another.

Reply:
Are we sure these fixtures are correct ? 
From what I tested, maxPrice is bouded to 0 >= n <= 1200000000 with max precision 99999999. 
As such, I am having trouble understanding what fffffffffffffff7ffffffffffffffff7f should represent

Reply:
please check commit 322be733b65cdec537df9a0777f4c4188edcfe28 to assess my question

Reply:
> Are we sure these fixtures are correct ?
> From what I tested, maxPrice is bouded to 0 >= n <= 1200000000 with max precision 99999999.
> As such, I am having trouble understanding what fffffffffffffff7ffffffffffffffff7f should represent

inconsistent implementation on DeFiCh/ain side I think `fffffffffffffff7ffffffffffffffff7f` is a value they use if no max value is defined.

Reply:
Oh I see, how should we handle it ?

Reply:
> Oh I see, how should we handle it ?

I think we can ignore this